source/_integrations/google_stackdriver.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Google Stackdriver"
+description: "Setup for Google Stackdriver integration"
+logo: google_stackdriver.png
+ha_category:
+  - History
+ha_release: 0.104
```

Comment:
`ha_iot_class` is missing.

Reply:
I am not sure what's the right class. Certainly cloud but neither push nor polling is true since data (logs not even entity related data) only flows from Home Assistant to Stackdriver so no need to push or poll. The logs are transfered in batches so the right thing would be "batched push" but will change it to "Cloud Push".